test/integration/TTLSecondsAfterFinished-in-jobs/00-install.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: lifecycle.keptn.sh/v1alpha3
```

Comment:
I am missing a check (00-assert.yaml file) here, can you please add it? it should contain the job anc a check that the TTL was propagated to job

Reply:
yes, got that